go.sum

```diff
@@ -570,8 +570,12 @@ github.com/kylelemons/godebug v1.1.0 h1:RPNrshWIDI6G2gRW9EHilWtl7Z6Sb1BR0xunSBf0
 github.com/kylelemons/godebug v1.1.0/go.mod h1:9/0rRGxNHcop5bhtWyNeEfOS8JIWk580+fNqagV/RAw=
 github.com/kyma-project/runtime-watcher/listener v0.0.0-20231011102033-b8383d92883e h1:yJoi2kUTS1CKOBQOG8BRNxWAdEAk63DurKSJif6miUU=
 github.com/kyma-project/runtime-watcher/listener v0.0.0-20231011102033-b8383d92883e/go.mod h1:h+qRkxtzjaBmmwNW8om8othH+xCm4iK4a53WoMoMe50=
```

Comment:
You need to run go mod tidy to remove the old listener and template-operator versions here